src/requestLib.js

```diff
@@ -86,17 +98,18 @@ var RequestLib = {
             });
           }
         });
-  
+
         clientRequest.on('error', function (err) {
           request.end();
           reject({
             message: err,
             customTopic: constants.TOPICS.CLIENT_REQUEST_WITH_RETRIES + retries
           });
         });
-  
+
         clientRequest.on('end', function () {
-          RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_END, params.request.method + ' ' + params.request.url, false, {
+          var requestUrl = serverType === constants.SERVER_TYPES.PROXY ? clientRequest.url : "http://" + clientRequest.headers.host + requestOptions.path;
+          RdGlobalConfig.reqLogger.info(constants.TOPICS.CLIENT_REQUEST_END, params.request.method + ' ' + requestUrl, false, {
```

Comment:
need to do similar changes of logging at line 142 as well.

Reply:
Reason: The tool request logging logs the scheme as HTTP...
![image](https://user-images.githubusercontent.com/22256912/96971513-d588ea00-1532-11eb-8b6a-3c1587299ec6.png)

Reply:
verify if there are any other such cases...

Reply:
can be resolved...